### ISSUE TITLE
Simplify repos in prep for hosting

### ISSUE THREAD

**Issue by jadudm** [CMSgov/CMCS-DSG-DSS-Oversight#246]
[ ] Docker container simplification
[ ] Content upload? Automation via actions?
[ ] CI/CD with link checking?

All of this is contingent on understanding how the site will be hosted, and what is then possible given the time available. Therefore, this is largely a hypothetical, and we should take this ticket to mean "try and make sure things are as simple as possible, but no simpler, as we move to hosting the site with CMS."

**Comment by jadudm** [comment]
As a first step, the "simplify" branch was merged back into "staging." 

1. `staging` was merged into `simplify` to bring `simplify` up to date with content/edits.
2. `simplify` was merged back into `staging`.
3. Old things were removed from `staging`.

I don't believe we lost content or edits as a result, as this dance should have preserved all of our work on `staging`. It is now slimmer, renders much faster, and is ready for further simplification.

**Comment by jadudm** [comment]
Adding notes regarding general cleanup...

Added a bit of CSS for handling print conditions.

**Comment by jadudm** [comment]
There are many things in the air regarding hosting. For now, this ticket is being closed in favor of future, more specific tickets.
